homeassistant/components/light/tradfri.py

```diff
@@ -30,6 +30,58 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     lights = [dev for dev in devices if dev.has_light_control]
     add_devices(Tradfri(light) for light in lights)
 
+    groups = gateway.get_groups()
+    lights = [dev for dev in groups]
+    add_devices(TradfriGroup(light) for light in lights)
+
+
+class TradfriGroup(Light):
+    """The platform class required by hass."""
+
+    def __init__(self, light):
+        """Initialize a Light."""
+        self._light = light
+
+        self._light_control = light
+        self._light_data = light
+        self._name = light.name
+        self._features = SUPPORT_BRIGHTNESS
```

Comment:
No need to initiate here since it's hard-coded. Just return `SUPPORT_BRIGHTNESS` from the property.